tests/build/build_templates.go

```diff
@@ -125,10 +140,28 @@ var _ = framework.BuildSuiteDescribe("Build templates E2E test", Label("build",
 			}
 		})
 
+		It(fmt.Sprintf("triggers PipelineRun for symlink component with source URL %s", pythonComponentGitSourceURL), Label(buildTemplatesTestLabel), func() {
```

Comment:
What is symlink component?

Reply:
We have named the component with git repo containing symlink outside the repository as symlinkComponent [here](https://github.com/redhat-appstudio/e2e-tests/blob/6fff2460639059b0cb041bacbc37c0c01558ee59/tests/build/build_templates.go#L102-L111) , the git-clone step of the build PLR will be failed with error `cloned repository contains symlink pointing outside of the cloned repository`